securesystemslib/ecdsa_keys.py

```diff
@@ -121,6 +125,10 @@ def generate_public_and_private(scheme='ecdsa-sha2-nistp256'):
     'securesystemslib.formats.PEMECDSA_SCHEMA', respectively.
   """
 
+  if not CRYPTO: # pragma: no cover
```

Comment:
This is technically easily handled thanks to the versatility of coverage[[1]](https://coverage.readthedocs.io/en/coverage-5.0.3/config.html#config), but I held off on doing it because I wasn't sure how valuable coverage results would be for the purepy environments - at least in the current setup with only `check_public_interfaces.py` run in the purepy Tox environments we won't cover much of the code at all.

Reply:
I was thinking of a setup with mutual exclusive coverage exemption. Something like a `# pragma: purepy`, to mark lines/branches that are excluded from `py{27, 35, 36, 37, 38}`, but are also the only lines/branches considered for `purepy{27, 38}`, so that all lines must be tested by either one or the other. But let's do this in a separate PR (I'll add a note on #179).